README.md

```diff
@@ -34,7 +34,7 @@ antigen bundle jessarcher/zsh-artisan
 First download the plugin to your oh-my-zsh custom plugin location:
 
 ```zsh
-git clone https://github.com/jessarcher/zsh-artisan.git ~/.oh-my-zsh/custom/plugins/artisan
+git clone https://github.com/jessarcher/zsh-artisan.git ${ZSH_CUSTOM1:-$ZSH/custom}/plugins/artisan
```

Comment:
Thanks for the PR!

I'm curious about `$ZSH_CUSTOM1` - is the `1` intentional? It's unset on my system, but `$ZSH_CUSTOM` _is_ set..

Reply:
@jessarcher $ZSH_CUSTOM works for me. Thanks for the plugin!